core/src/test/java/org/infinispan/distribution/ZeroCapacityNodeTest.java

```diff
@@ -149,6 +156,54 @@ private void createCache(ConfigurationBuilder cb, String cacheName) {
       waitForClusterToForm(cacheName);
    }
 
+   public void testZeroCapacityFactorNodeStartsFirst() throws Exception {
+      ConfigurationBuilder cb = new ConfigurationBuilder();
+      cb.clustering().cacheMode(CacheMode.DIST_SYNC);
+      cb.clustering().hash().numSegments(NUM_SEGMENTS);
+      cb.clustering().remoteTimeout(100);
+
+      ConfigurationBuilder cbZero = new ConfigurationBuilder();
+      cbZero.clustering().cacheMode(CacheMode.DIST_SYNC);
+      cbZero.clustering().hash().numSegments(NUM_SEGMENTS);
+      cbZero.clustering().hash().capacityFactor(0f);
+      cbZero.clustering().remoteTimeout(100);
```

Comment:
No need for multiple lines for each of the attributes, it can be chained as a single for both `cb` and `cbZero`.